packages/select/src/MenuButton.tsx

```diff
@@ -81,30 +183,28 @@ const menuButtonTextStyle = css`
   max-width: 100%;
 `;
 
-const errorColor: Record<Mode, string> = {
-  [Mode.Light]: palette.red.base,
-  [Mode.Dark]: '#F97216',
+const errorColor: Record<Theme, string> = {
+  [Theme.Light]: palette.red.base,
+  [Theme.Dark]: '#FF6960', // TODO: palette.red.light1 is different in figma(#FF6960) which does not match react(#EF5752) but will be updated with this ticket https://jira.mongodb.org/browse/PD-2206
```

Comment:
I feel like we should write `red.light1` here, and have this automatically update when palette gets updated. Otherwise we may forget to come back and add the token